server/portal/apps/accounts/managers/accounts.py

```diff
@@ -82,6 +82,16 @@ def setup(username, system):
     user = check_user(username)
     mgr = UserSystemsManager(user, system)
     home_dir = mgr.get_private_directory(user)
+    systemId = mgr.get_system_id()
+    system = StorageSystem(user.agave_oauth.client, id=systemId)
+    success, result = system.test()
+    if success:
+        logger.info(
+            "{username} has valid configuration for {systemId}, skipping creation".format(
+                username=username, systemId=systemId
+            )
+        )
+        return home_dir, success
```

Comment:
here the tuple that is returned is ```home_dir, success``` but the returned tupe is ```home_dir, home_sys``` in line 97.  

what about making this match 97 [see suggested change]?  or maybe returning nothing as I don't think its used.
```suggestion
        return home_dir, system
```

Reply:
> KeyServiceCreationStep is needed for the _PORTAL_USER_ACCOUNT_SETUP_STEPS in `Testing Key Service based System Creation `, right?

Yes. The above config should have everything you need.

Reply:
Removed the second result in the tuple. I think the home_dir result is still unused but I don't want to completely eliminate the result, just in case.